notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -23,8 +23,18 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def get_victim_active_ranges(df: pd.DataFrame) -&gt; dict[str, tuple[int, int]]:
</pre><p>is this and get_victim_change_steps just copied from win-rate-vs-training.steps.ipynb? we can move them out to <code>utils.py</code> to avoid code duplication </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
i went ahead and made this change myself to this branch since Adam had suggested moving these functions out of win-rate-vs-training-steps.ipynb to me a while back